internal/core/src/segcore/segcore_init_c.cpp

```diff
@@ -26,10 +28,23 @@ SegcoreInitImpl() {
     eg::KnowhereConfig::SetStatisticsLevel(0);
     el::Configurations el_conf;
     el_conf.setGlobally(el::ConfigurationType::Enabled, std::to_string(false));
+
+    // initializing segcore config
+    try {
+        SegcoreConfig& config = SegcoreConfig::default_config();
+        if (config_dir != NULL) {
+            std::string config_file = std::string(config_dir) + "advanced/segcore.yaml";
+            config.parse_from(config_file);
+            std::cout << "Parse config file: " << config_file << ", chunk_size: " << config.get_size_per_chunk()
```

Comment:
std::cout? we don't have any logger at cpp level?

Reply:
Yes, segcore has no logger